### PR TITLE
feat: Add CasparCG cron job system setting

### PR DESCRIPTION
**Pull request by ianshade**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
This PR adds the following feature: A Core System setting to disable the cron job restarting CasparCG servers.

* **What is the current behavior?** (You can also link to an open issue here)
The cron job that restarts CasparCG servers is running every night and there is no way to disable it.

* **What is the new behavior (if this is a feature change)?**
A section called _Cron jobs_, with a setting called _Disable CasparCG restart job_ is added to Core System Settings. The setting is off by default. When the setting is on, CasparCG servers will not be restarted during the execution of the nightly cron jobs.

* **Other information**:

**Status**

- [ ] Code documentation for the relevant parts in the code have been added/updated by the PR author
- [x] The functionality has been tested by the PR author
- [ ] The functionality has been tested by NRK

### CODE REVIEW

File: meteor/lib/collections/CoreSystem.ts [hunk 1]
```diff
@@ -89,6 +89,13 @@ export interface ICoreSystem {
 		transactionSampleRate?: number
 	}
 	enableMonitorBlockedThread?: boolean
+
+	/** Cron jobs running nightly */
+	cron?: {
+		casparCG?: {
```
Comment (Julusian) [inline meteor/lib/collections/CoreSystem.ts:95]: it would be clearer to call this `casparCGRestart`

Comment (ianshade) [reply meteor/lib/collections/CoreSystem.ts]: I renamed it to `casparCGRestart`

**Comment by ianshade** [comment]
I refactored the setting to say _Enable CasparCG restart job_. It will be on by default. A migration was added.

**Comment by sparkpunkd** [comment]
This is relevant to NRK RK5/6 workflows so I would like to get this included soon.

**Pull request merged**